qiskit_acqua/ising/maxcut.py

```diff
@@ -156,28 +156,24 @@ def sample_most_likely(n, state_vector):
     """Compute the most likely binary string from state vector.
```

Comment:
@chunfuchen the param `n` is kept by mistake, right? it's dropped for the others

Reply:
yes, my mistake.